.github/workflows/e2e-component-pipeline.yml

```diff
@@ -1,171 +1,191 @@
 ---
 name: Component-Pipeline
 on:
-  workflow_dispatch:
-    inputs:
-      e2eTestImage:
-        default: 'litmuschaos/litmus-e2e:ci'
-      goExperimentImage:
-        default: 'litmuschaos/go-runner:ci'
-      newLibImage:
-        default: 'litmuschaos/go-runner:ci'
-      oldLibImage:
-        default: 'litmuschaos/go-runner:latest'        
-      operatorImage:
-        default: 'litmuschaos/chaos-operator:ci'
-      runnerImage:
-        default: 'litmuschaos/chaos-runner:ci'
-      chaosNamespace:
-        default: 'litmus'
-      imagePullPolicy:
-        default: 'Always'
-      experimentImagePullPolicy:
-        default: 'Always'
-      updateWebsite: 
-        default: 'false'
-    
+  schedule:
+    - cron: "30 20 * * *" # Daily 02:30 AM in midnight
+
 defaults:
   run:
-    working-directory: /go/src   
-      
+    working-directory: /go/src
+
 jobs:
+  ### Setup Variables
+
+  Initialize_Variables:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./ ## We are not using container here
+    outputs:
+      e2eTestImage: "litmuschaos/litmus-e2e:ci"
+      goExperimentImage: "litmuschaos/go-runner:ci"
+      newLibImage: "litmuschaos/go-runner:ci"
+      oldLibImage: "litmuschaos/go-runner:latest"
+      operatorImage: "litmuschaos/chaos-operator:ci"
+      runnerImage: "litmuschaos/chaos-runner:ci"
+    steps:
+      - run: |
+          echo "Setting the variables for other Jobs"
 
```

Comment:
Let keep it simple, we can provide only e2e image(in every job) and exp image. Also runner and operator image( we are using this in installation only).